modules/mbedtls/configs/config-tls-generic.h

```diff
@@ -438,6 +438,10 @@
 #include CONFIG_MBEDTLS_USER_CONFIG_FILE
 #endif
 
+#if defined(CONFIG_NRF_CC3XX_PLATFORM)
+#define MBEDTLS_PLATFORM_ZEROIZE_ALT
+#endif
+
```

Comment:
Strange. I thought we generated the config.

It's not clear to me when this file is used.

But perhaps we can add the MBEDTLS_PLATFORM_ZEROIZE_ALT define from the nrf build system so we avoid this patch.

It should also be in it's own commit.

Reply:
The difference is in the new cc3xx runtime library which has this symbol

Reply:
I don't know what that means, but let's add this to the task we already have about removing noups to the Zephyr repo.